src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/PackageManagerCloseEvent.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using NuGet.Common;
+
+namespace NuGet.PackageManagement.VisualStudio
+{
+    public sealed class PackageManagerCloseEvent : TelemetryEvent
+    {
+        private const string EventName = "PMUIClose";
+
+        public PackageManagerCloseEvent(
+            Guid parentId,
+            bool isSolutionLevel,
+            string tab,
```

Comment:
Try to use explicit type to only emit event for a valid PM UI tab.

```suggestion
            ItemFilter tab,
```

Reply:
Discussed offline. It follows the pattern of other telemetry events such as the PMUIRefresh event. We can consider updating later.